parlai/mturk/tasks/controllable_dialogue/run.py

```diff
@@ -0,0 +1,310 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+from parlai.core.params import ParlaiParser
+from parlai.core.agents import create_agent
+from parlai.mturk.core.mturk_manager import MTurkManager
+import parlai.mturk.core.mturk_utils as mturk_utils
+
+from worlds import ControllableDialogEval, PersonasGenerator, PersonaAssignWorld
+from task_config import task_config
+import model_configs as mcf
+
+from threading import Lock
+import gc
+import datetime
+import json
+import logging
+import os
+import sys
+import time
+import copy
+import random
+import pprint
+
+
+# Place the WorkerID of anyone you want to ban in here
+BLOCKLIST = "".strip().split()
```

Comment:
can we get rid of this and references to it?

Reply:
sure